blue_hardware_drivers/src/BLDCDriver_communication_freq_test.cpp

```diff
@@ -105,12 +85,45 @@ int main(int argc, char **argv) {
       }
     }
   }
+}
 
+int main(int argc, char **argv) {
+  ros::init(argc, argv, "comms", ros::init_options::AnonymousName);
+
+  ros::NodeHandle n;
+  std::vector<comm_id_t> board_list;
+  std::map<comm_id_t, std::vector<double> > velocity_history_mapping;
+
+
+  char* port = argv[1];
+  char* board_nums = argv[2];
+  char* enumerate = argv[3];
+
+  std::string num = "";
+  for (unsigned int i = 0; board_nums[i] != '\0'; i++) {
+    if (board_nums[i] == ',')
+      continue;
+    comm_id_t id_num = std::atoi(board_nums + i);
+    board_list.push_back(id_num);
+  }
```

Comment:
if you have time ( I can also help if you don't ), I think it's worth considering standardizing some argparse-style library for handling CLI flags; would help us with generating help text, somewhat less sketchily handling this conversion stuff, validating inputs, etc

examples:
- [CLI11s](https://github.com/CLIUtils/CLI11) (popular, familiar API if you've been using `argparse` in Python)
- [abseil](https://abseil.io/docs/cpp/guides/flags) (Google-y and neat, mostly designed for bigger repos/scattered global settings though)

Reply:
Feel free to do this.